backend/requirements.txt

```diff
@@ -1,20 +1,20 @@
 anyio==3.6.1
 click==8.1.3
-fastapi==0.79.0
+fastapi==0.89.1
```

Comment:
@alexgarel Hopefully all the necessary dependencies for fastAPI and other packages has also been updated.

Reply:
seems so :-)